src/modules/tests/test-data/cat-be/test-data.cat-be.selector.ts

```diff
@@ -0,0 +1,119 @@
+import { Competencies, LegalRequirements } from '../test-data.constants';
+import { get } from 'lodash';
+import { CompetencyOutcome } from '../../../../shared/models/competency-outcome';
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { CatBELegalRequirements } from '../test-data.models';
+import { QuestionProvider } from '../../../../providers/question/question';
+import { TestCategory } from '../../../../shared/models/test-category';
+import { VehicleChecksQuestion } from '../../../../providers/question/vehicle-checks-question.model';
+
+export const getDrivingFaultCount = (
+    data: CatBEUniqueTypes.TestData, competency: Competencies) => data.drivingFaults[competency];
+
+export const getManoeuvres = (data: CatBEUniqueTypes.TestData): CatBEUniqueTypes.Manoeuvres => data.manoeuvres;
+
+export const hasManoeuvreBeenCompleted = (data: CatBEUniqueTypes.TestData) => {
+  return (
+    get(data.manoeuvres, 'reverseLeft.selected')
+  );
+};
+
+export const hasEyesightTestGotSeriousFault = (data: CatBEUniqueTypes.TestData) => data.eyesightTest.seriousFault;
+
+export const hasLegalRequirementBeenCompleted = (
+  data: CatBEUniqueTypes.TestRequirements, legalRequirement: LegalRequirements) => {
+  return data[legalRequirement];
+};
+
+export const getVehicleChecks = (
+  state: CatBEUniqueTypes.TestData): CatBEUniqueTypes.VehicleChecks => state.vehicleChecks;
+
+export const getTellMeQuestion = (state: CatBEUniqueTypes.VehicleChecks): VehicleChecksQuestion => {
+  const questionProvider: QuestionProvider =  new QuestionProvider();
+  return questionProvider
+    .getTellMeQuestions(TestCategory.BE)
+    .find(question => question.code === get(state, 'tellMeQuestion.code'));
+};
```

Comment:
This may need looking at as the show me tell me questions has been merged in to latest develop.
For BE we can have multiple show me and tell me questions - so it may be that getTellMeQuestion will need replacing with something that gets a specific question by index or returns an array of questions.

Reply:
OK, is that something to look at this ticket?

Reply:
I think we can revisit it later - we are awaiting a bit of work around the vehicle checks so will have to incorporate that. So could do it then

Reply:
Cool